conans/util/locks.py

```diff
@@ -61,6 +61,10 @@ def _readers(self):
         try:
             return int(load(self._count_file))
         except IOError:
+            self._output.info("Could not open %s!" % self._count_file)
+            return 0
+        except UnicodeEncodeError:
```

Comment:
Are you sure about this? The problem when reading something different than an ``int`` is a ``ValueError``. I wouldn't try to ``load()`` again the contents of the file, if it failed the first time, it will fail again. The ``UnicodeEncodeError`` can come from the ``load()`` not the ``int()``.

I would be fine with doing an ``except IOError, UnicodeEncodeError, ValueError`` to cover all possible cases.